File: CS/Dashboard_LoadingAndSaving/Form1.Internal.cs
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.LookAndFeel;

namespace Dashboard_LoadingAndSaving {
    public partial class Form1 {
        public DialogResult InvokeMessageBox() {
            return XtraMessageBox.Show(new UserLookAndFeel(dashboardDesigner1), 
                                       "Do you want to save the dashboard to the default location?", 
                                       "Save As", 
                                       MessageBoxButtons.OKCancel);
        }
    }
}

File: VB/Dashboard_LoadingAndSaving/Form1.Internal.vb
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.LookAndFeel

Namespace Dashboard_LoadingAndSaving

    Public Partial Class Form1

        Public Function InvokeMessageBox() As DialogResult
            Return XtraMessageBox.Show(New UserLookAndFeel(dashboardDesigner1), "Do you want to save the dashboard to the default location?", "Save As", MessageBoxButtons.OKCancel)
        End Function
    End Class
End Namespace
